### PR TITLE
docs: add examples

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

- Add example for creating, waiting and getting the connection for a cluster
- Add example fro creating recommended indexes

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: examples/atlas/create-cluster.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+```bash
+mongocli atlas clusters create MyCluster \
+  --region EASTERN_US \
+  --members 3 \
+  --tier M30 \
+  --provider GCP \
+  --mdbVersion 4.4 \
+  --diskSizeGB 30 && \
+mongocli atlas clusters watch MyCluster && \
+mongocli atlas clusters describe MyCluster -o go-template="{{.SrvAddress}}"
```
Comment (andreaangiolillo) [inline examples/atlas/create-cluster.md:14]: [q] what about inserting the definition of each flag, something like this
```suggestion
mongocli atlas clusters create MyCluster \
  --region EASTERN_US \ # Physical location of your MongoDB cluster.
  --tier M30 \ # Tier for each data-bearing server in the cluster. 
  --provider GCP \ # Name of your cloud service provider.
  --mdbVersion 4.4 \ # MongoDB version of the cluster to deploy. 
  --diskSizeGB 30 && \ # Capacity, in gigabytes, of the host’s root volume.
mongocli atlas clusters watch MyCluster && \
mongocli atlas clusters describe MyCluster -o go-template="{{.SrvAddress}}"
```

File: examples/atlas/create-cluster.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Create a Cluster
+
+To create and get the connection string for a cluster you can do:
+
+```bash
+mongocli atlas clusters create MyCluster \
+  --region EASTERN_US \
+  --members 3 \
```
Comment (andreaangiolillo) [inline examples/atlas/create-cluster.md:8]: [nit] the default value of `--members` is 3 so you can remove this

**Review by andreaangiolillo — APPROVED** [review]
Looks good! I have 2 suggestions :)

File: examples/atlas/create-cluster.md (hunk 1 above)
Comment (gssbzn) [reply examples/atlas/create-cluster.md:14]: this was a nice idea but I just tried to copy/paste the new command and it no longer works so I won't add this

File: examples/atlas/create-cluster.md (hunk 2 above)
Comment (gssbzn) [reply examples/atlas/create-cluster.md:8]: I like to have examples showing it so people know it exists

**Pull request merged**